compose/composer.go

```diff
@@ -206,15 +223,10 @@ func executeFlow(ctx context.Context, taskC chan Task, resultsChan chan Result,
 	}
 }
 
-func (c *Composer) PrintConfig() {
-	fmt.Println("##########################")
-	fmt.Println("# Compose Configuration #")
-	fmt.Println("##########################")
-	fmt.Printf("ConfigFile: %s\n", c.ConfigFile)
-	if c.CherryPickedFlow != "" {
-		fmt.Printf("Selected Flow: %s\n", c.CherryPickedFlow)
+// TODO: add unit test for this
+func getMaxNumberOfWorkers(totalFlows, numberFromFlag int) int {
+	if numberFromFlag == 0 || totalFlows <= numberFromFlag {
+		return totalFlows
```

Comment:
Renaming few things just for readability.
```suggestion
func getWorkersCount(flowsCount, countFromFlag int) int {
	if countFromFlag <= 0 || countFromFlag > flowsCount {
		return flowsCount
	}
	return countFromFlag
}
```